### PR TITLE
MAVlink forwarding

### PR DESCRIPTION
**Pull request by LorenzMeier**
@sanderux This hopefully fixes the issues you were facing with forwarding. Could you cross-test and fix where required?

### CODE REVIEW

**Comment by sanderux** [comment]
nice, i'll have a look

**Comment by sanderux** [comment]
I added changes that allow relaying when it's from the same systemID but a different component. This is required to relay statustext messages and have them show up in the GCS

**Comment by sanderux** [comment]
Tested, works

File: src/modules/mavlink/mavlink_main.cpp [hunk 1]
```diff
@@ -1999,6 +2006,9 @@ Mavlink::task_main(int argc, char *argv[])
 		configure_stream("NAMED_VALUE_FLOAT", 1.0f);
 		configure_stream("VFR_HUD", 4.0f);
 		configure_stream("WIND_COV", 1.0f);
+		// Image captured runs at full rate independent of
+		// any limit
+		configure_stream("CAMERA_IMAGE_CAPTURED", 1.0f);
```
Comment (LorenzMeier) [inline src/modules/mavlink/mavlink_main.cpp]: I need to remove this - this doesn't work without further changes.

File: src/modules/mavlink/mavlink_main.cpp [hunk 2]
```diff
@@ -2030,9 +2040,12 @@ Mavlink::task_main(int argc, char *argv[])
 		configure_stream("SYSTEM_TIME", 1.0f);
 		configure_stream("TIMESYNC", 10.0f);
 		configure_stream("CAMERA_CAPTURE", 2.0f);
-		//camera trigger is rate limited at the source, do not limit here
-		configure_stream("CAMERA_TRIGGER", 500.0f);
-		configure_stream("CAMERA_IMAGE_CAPTURED", 5.0f);
+		// Camera trigger runs at full rate independent of
```
Comment (LorenzMeier) [inline src/modules/mavlink/mavlink_main.cpp]: I need to revert this - this doesn't work without further changes.

File: src/modules/mavlink/mavlink_main.cpp [hunk 3]
```diff
@@ -2084,8 +2097,12 @@ Mavlink::task_main(int argc, char *argv[])
 		configure_stream("NAMED_VALUE_FLOAT", 50.0f);
 		configure_stream("VFR_HUD", 20.0f);
 		configure_stream("WIND_COV", 10.0f);
-		configure_stream("CAMERA_TRIGGER", 500.0f);
-		configure_stream("CAMERA_IMAGE_CAPTURED", 5.0f);
+		// Camera trigger runs at full rate independent of
```
Comment (LorenzMeier) [inline src/modules/mavlink/mavlink_main.cpp]: I need to revert this - this doesn't work without further changes.

**Comment by sanderux** [comment]
@LorenzMeier can you revert your camera changes so we can get this in?

**Comment by LorenzMeier** [comment]
Good to merge. I also verified memory footprint and made sure UAVCAN setups should still be fine.

**Comment by LorenzMeier** [comment]
Jenkins test this please.

**Pull request merged**